bio2zarr/vcf2zarr/vcz.py

```diff
@@ -1248,7 +251,7 @@ def mkschema(
 
 
 def encode(
```

Comment:
These high-level methods are breaking the clarity of the division of labour here; it is probably too much to move them out in this PR, but we should look at moving them somewhere else. The current modules should just be concerned with writing VCZ efficiently, and not know any thing about where that data comes from.